app/views/escorts/_actions.html.slim

```diff
@@ -5,5 +5,8 @@ ul.actions
   - elsif escort.completed?
     li
       = link_to_print_in_new_window(escort, styles: 'button button-print')
+  - unless escort.issued? || escort.cancelled?
```

Comment:
🙅 This is exactly the reason why we shouldn't keep on repeating logic all over the application that should be in the escort. This logic already exists in the escort controller. Can we please follow this [comment](https://github.com/ministryofjustice/moving-people-safely/pull/337/files#r125505777) suggestion pls to DRY this?